src/OpenLoco/src/S5/S5.cpp

```diff
@@ -613,6 +614,47 @@ namespace OpenLoco::S5
         return importSaveToGameState(fs, flags);
     }
 
+    // struct NonCustomObjectHeaderHash
+    //{
+    //     template<class T>
+    //     static inline void hashCombine(std::size_t& seed, const T& v)
+    //     {
+    //         std::hash<T> hasher;
+    //         seed ^= hasher(v) + 0x9e3779b9 + (seed << 6) + (seed >> 2); // Based on boost::hash_combine
+    //     }
+
+    //    std::size_t operator()(ObjectHeader const& header) const noexcept
+    //    {
+    //        std::size_t h = 0U;
+    //        hashCombine(h, header.flags & 0xFFU);
+    //        hashCombine(h, header.getName());
+    //        return h;
+    //    }
+    //};
+
+    static const sfl::static_unordered_flat_map<ObjectHeader, ObjectHeader, 7> _assetPack = {
```

Comment:
can't decide if using a sfl::static_unordered_flat_map or a std::unordered_map is better. For now with so few entries i assume sfl is better but as it doesn't use a hash i assume it drops off in performance at higher sizes.